test/elife-loader.spec.js

```diff
@@ -0,0 +1,92 @@
+const chai = require('chai');
+const sinon = require('sinon');
+
+const expect = chai.expect;
+const spy = sinon.spy;
+
+// load in component(s) to be tested
+let loader = require('../assets/js/elife-loader');
+
+describe('The eLife Loader', function () {
+  "use strict";
+
+  let loader;
+
+  describe("checks if the network information", function() {
```

Comment:
`describe` ideally should describe a thing, for example a function. So here it might be clearer to say:

```
describe('the isNetworkInformationAvailable() function', function () {
```

and then the tests within could be described in terms of:

```
  it('returns true when the network information is available'...);
  it('returns false when the network information is not available'...);
```

The advantage of this type of wording is that when a test fails, it's immediately clear which function has the problem, and which of its behaviours is breaking.

Reply:
Completed